### DISCUSSION TITLE
Is there any feature coming or is available in flow launcher that allows us to search windows settings?

### DISCUSSION THREAD

**Discussion by rsmyst** [Flow-Launcher/Flow.Launcher#666] [Q&A] [answered]
Like a feature where I can search in wifi, and it brings up wifi and connectivity settings from windows 10's settings panel

**Comment by jjw24** [comment, marked as answer]
Hi @rsmyst, yes This feature is already in the works:
 https://github.com/Flow-Launcher/Flow.Plugin.WindowsSettings
